### PR TITLE
Add New Spider templates with automatic labelling for the well known storefinders

### LINKED ISSUE #9346: Explore ways to automate labelling?
@davidhicks does a lot of good issue curation and labelling.

For those without commit access, it's a bit trickier - have to manipulate titles.

Ideally, it a label-bot could recognize patterns like  "has Structured Data" or "uses XYZSpider" and auto label.

https://github.com/facelessuser/label-bot I have seen with mixed results and it's a bit annoying.
https://github.com/marketplace/issue-labeler-bot maybe?
https://github.com/actions/labeler for pull requests, though that is much less of an issue

### PR DESCRIPTION
**Pull request by CloCkWeRX**
Related:

- https://github.com/alltheplaces/alltheplaces/pull/9456
- Fix #9346  ?

To test:
https://github.com/CloCkWeRX/alltheplaces/issues/new/choose

TODO:

- [x] Check all the labels are correct
- [x] For the spiders that take API keys, add a question
- [x] Add any other helpful documentation

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/spider_request_stockinstore.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: "New Spider: Stockinstore"
```
Comment (CloCkWeRX) [inline .github/ISSUE_TEMPLATE/spider_request_stockinstore.yml:1]:
```suggestion
name: "New Spider: stockinstore"
```

File: .github/ISSUE_TEMPLATE/spider_request_stockinstore.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+description: A spider using the Stockinstore storefinder
+labels: "Store finder: Stockinstore"
```
Comment (CloCkWeRX) [inline .github/ISSUE_TEMPLATE/spider_request_stockinstore.yml:3]:
```suggestion
labels: "Store finder: stockinstore"
```

File: .github/ISSUE_TEMPLATE/spider_request_stockinstore.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+name: "New Spider: Stockinstore"
+description: A spider using the Stockinstore storefinder
```
Comment (CloCkWeRX) [inline .github/ISSUE_TEMPLATE/spider_request_stockinstore.yml:2]:
```suggestion
description: A spider using the stockinstore storefinder
```

File: .github/ISSUE_TEMPLATE/spider_request_freshop.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+name: "New Spider: Freshop"
+description: A spider using the Freshop storefinder. Makes requets to `https://api.freshop.com/1/stores?app_key={app_key}`
```
Comment (CloCkWeRX) [inline .github/ISSUE_TEMPLATE/spider_request_freshop.yml:2]:
```suggestion
description: A spider using the Freshop storefinder. Makes requests to `https://api.freshop.com/1/stores?app_key={app_key}`
```

File: .github/ISSUE_TEMPLATE/spider_request_rexel.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+name: "New Spider: Rexel"
+description: A spider using the Rexel storefinder. Makes calls to `/store-finder/findNearbyStores`
```
Comment (CloCkWeRX) [inline .github/ISSUE_TEMPLATE/spider_request_rexel.yml:2]:
```suggestion
description: A spider using the Rexel storefinder, typically part of the Rexel Group. Makes calls to `/store-finder/findNearbyStores`
```

File: .github/ISSUE_TEMPLATE/spider_request_freshop.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Wikidata ID
+      placeholder: Q38076
+      description: >-
+        If possible get the ID from
+        [Wikidata](https://www.wikidata.org/wiki/Wikidata:Main_Page)
+- type: input
```
Comment (CloCkWeRX) [inline .github/ISSUE_TEMPLATE/spider_request_freshop.yml:18]:
```suggestion
  - type: input
```

**Comment by iandees** [comment]
This is a pretty overwhelming list to present to people who want to file an issue.

**Comment by CloCkWeRX** [comment]
Yeah; trying to strike a balance between the internal people and newcomers is a bit tricky.

The advantage of this, though it forces you to go down a certain pathway is it will prompt you with a lot of questions or specific detail for that kind of spider; making it easier to write.

**Pull request closed without merging**